packages/core/src/queries/user.ts

```diff
@@ -137,3 +137,14 @@ export const clearUserCustomDataById = async (id: string) => {
     throw new UpdateError(Users, { set: { customData: {} }, where: { id } });
   }
 };
+
+export const deleteUserIdentity = async (userId: string, connectorId: string) => {
+  const { identities } = await findUserById(userId);
+
+  return pool.one<User>(sql`
+    update ${table}
+    set ${fields.identities}=${JSON.stringify(identities)}::jsonb-${connectorId}
```

Comment:
Can we delete the property directly from db's data? Current implementation is "query" and "save".

Reply:
An example
```
UPDATE my_table
SET data = data::jsonb - 'attrB'
WHERE id = 'A';
```

Reply:
do we need to check ahead if identities[connectorId] exist?

Reply:
In the current use case, we checked whether the connector exists in identities. In general cases, the SQL command will not throw unexpected errors per offline discussion.